doc/diagrams/c4-model/model.dsl

```diff
@@ -51,5 +51,25 @@ measurementsDomain = group "Measurements Domain" {
 }
 
 transferAgreementsDomain = group "Transfer Agreements Domain" {
+    taApi = container "Transfer Agreement API" "" ".NET Web Api" {
+        connectionsController = component "Connections Controller" "Allows users to see connections of their company." ".NET Web Api Controller"
+        connectionInvitesController = component "Connection-Invites Controller" "Allows users to create connections with other companies" ".NET Web Api Controller"
+        TransferAgreementsController = component "Transfer Agreements Controller" "Allows users to create transfer agreements with other companies" ".NET Web Api Controller"
+        TransferAgreementAutomation = component "Transfer Agreements Automation" "Transfers certificates within a given transfer agreement" ".NET BackgroundService" {
+            this -> po "Transfers certificates"
```

Comment:
There is no system called "Project Origin" - it should be a system for Registry and one for Wallet System